yarpl/include/yarpl/Refcounted.h

```diff
@@ -176,35 +181,124 @@ bool operator==(const Reference<T>& lhs, const Reference<U>& rhs) noexcept {
   return lhs.get() == rhs.get();
 }
 
+template <typename T>
+bool operator==(const Reference<T>& lhs, std::nullptr_t) noexcept {
```

Comment:
Hmm calling `Reference<T>(std::nullptr_t)` should get inlined, but I'm fine with leaving this as is.